src/model/inc_classifier.py

```diff
@@ -64,15 +66,18 @@ def increment(self, new_classes: list[int]):
                 self.active_units[new_classes] = 1
 
         if old_nclasses != new_nclasses:
+            self.mul *= old_nclasses/new_nclasses
```

Comment:
This is scaled incorrectly. Please remove it and refer to the comment on line 79.